docs/scip/0003-roadmap.md

```diff
@@ -0,0 +1,79 @@
+# A roadmap for Scivision
+
+## Metadata
+
+Editors:
+  quantumjot, scotthosking, sebastianahnert (project PIs), ots22 (who
+  created this draft)
+
+Status (raw | draft | stable | deprecated | retired):
+  draft
+
+## Description
+
+This document summarises the main priorities of the Scivision team
+based at the Alan Turing Institute, for the current phase of the
+project supporting Scivision development, that started in January
+2023.
+
+Scivision originated at the Alan Turing Institute, and development is
+actively hosted there, but the project welcomes participation from
+anyone in the wider community. Any contributions related to
+Scivision's mission are gladly received, whether or not they relate to
+the items noted below.
+
+This roadmap is to be understood as listing areas that are likely to
+receive attention from the Turing team, and is provided in order to
+help users and contributors understand the direction of the project,
+given the Turing team's stake in it.
+
+Importantly, the roadmap doesn't commit anyone to any particular
+deliverables, there aren't specific dates associated with the items
+listed here, and it may be modified (it is likely to remain a
+perpetual 'draft').
+
+## Priority areas
+
+### Core features
+
+  - Data loaders (potentially replacing or supplementing Intake)
+  - Catalog schema improvements and model/datasource metadata; catalog
+    versioning
+  - Better specified requirements for models and datasources
+  - More robust handling of model dependencies
+
+### Community engagement
+
+  - Running events, including a regular community call
+  - Making Scivision a hub for best practices in sharing reproducible
+    CV models
+  - Informational videos (overview, getting started, task-focussed
+    tutorials)
+  - Direct support to researchers wishing to share their models and
+    datasets
+
+### Web interface
+
+  - Model/dataset matchmaking, graphical representation of models and
+    dataset interoperability
+  - Improved 'model cards'
```

Comment:
```suggestion
  - Improved 'model cards', detailing installation requirements and how to use CV models them within scivision 
```

Reply:
Are you happy with https://github.com/alan-turing-institute/scivision/blob/66b9e0a9afc916af6fd4f1abce9b0d54fe8d17ae/docs/scip/0003-roadmap.md?plain=1#L59-L61 ?